### ISSUE TITLE
Location/POI Database for Users and Fleets

### ISSUE THREAD

**Issue by mortik** [fleetyards/fleetyards#2645]
Add a database to create ingame "Waypoints" for Locations or **P**oints **o**f **I**nterest with information about how to get to the Waypoint, later maybe coordinates, interesting information about the POI etc etc.

**Comment by mortik** [comment]
**On hold — blocked by the game, not by us.**

A POI database is only useful if a location can make the round trip: recorded here, then acted on in-game. That second half does not exist today. Star Citizen has no coordinate system usable for this, and there is no way to set a waypoint from coordinates or an equivalent identifier. So we could store a POI, and nobody could do anything with it.

Worth separating the two halves of the original request, because they are not blocked to the same degree:

- **"how to get to the waypoint"** as written directions is buildable today — it is text attached to a named place, and needs nothing from the game.
- **coordinates** are the blocked half, and they are what makes the feature more than a notes app.

Building only the first half would ship a place to write down directions, which is a smaller and different product than what this issue asks for. That is a decision to make deliberately rather than by drifting into it.

**What would unblock this:** a stable, readable coordinate system in-game, plus some way to feed a waypoint back in. Until then there is nothing to design against — any data model picked now would be guessing at the shape of coordinates nobody has seen.

Keeping this open with `on-hold` rather than closing, since the request is sound and the obstacle is external.

🤖
